tools/rollup-es3/Tests/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+    "compilerOptions": {
+        "sourceMap": true,
+        "inlineSources": true,
+        "noImplicitAny": false,
+        "module": "amd",
+        "moduleResolution": "Node",
+        "target": "es5",
+        "alwaysStrict": true,
+        "declaration": true,
+        "lib": [ "dom", "es2015" ]
```

Comment:
are these required?

Reply:
Yes, the test project needs it's own tsconfig

Reply:
Is dom and es2015 required? This does not exist on other tsconfig in this repo

Reply:
Removed, but not sure while still listed here...